package.json

```diff
@@ -12,10 +12,12 @@
   "dependencies": {
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.8",
+    "lottie-react": "^2.4.0",
```

Comment:
I forgot to say earlier. You can uninstall dependencies you don't use with `npm uninstall.`

Reply:
@JessicaLoers well spotted! 👀 we left this one on purpose for future use, but thank you! 🤗